develop/plone/views/viewlets.rst

```diff
@@ -104,6 +104,10 @@ A viewlet manager can be rendered in a page template code using the following ex
 
   <div tal:replace="structure provider:viewletmanagerid" />
 
+.. note ::
```

Comment:
Spaces have meaning in reST. Minor syntax fix.

```suggestion
.. note::
```

Reply:
Done!